jwt-authorizer/src/layer.rs

```diff
@@ -156,22 +160,21 @@ where
     }
 }
 
-impl<ReqBody, S, C> Service<Request<ReqBody>> for AuthorizationService<S, C>
+impl<S, C> Service<Request<Body>> for AuthorizationService<S, C>
```

Comment:
This generic was removed, but I imagine this is important for a reason I cannot foresee. I need help determining the best course of action here...possibly bound on `hyper::body::Body` trait?